src/Compilers/CSharp/Test/Syntax/Parsing/LocalFunctionParsingTests.cs

```diff
@@ -2110,6 +2110,232 @@ void checkNodes()
             }
         }
 
+        [Fact, WorkItem(32106, "https://github.com/dotnet/roslyn/issues/32106")]
+        public void DuplicateAsyncs()
+        {
+            const string text = """
+                class Program
+                {
+                    void M()
+                    {
+                        async async void F1() { }
+                        async async async void F2() { }
+                        async async async async void F3() { }
+                        async async async async async void F4() { }
+                    }
+                }
+                """;
+
+            CreateCompilation(text).VerifyDiagnostics(
```

Comment:
>CreateCompilation(text).VerifyDiagnostics( [](http://example.com/codeflow?start=12&length=42)

Why this check is relevant?

Reply:
"Duplicate '{0}' modifier" is produced during binding, so we cannot fully test it with only syntax checks